blocks/code/package.json

```diff
@@ -12,7 +12,7 @@
   "author": "HASH",
   "scripts": {
     "build": "block-scripts build",
-    "codegen": "block-scripts codegen",
+    "generate-types": "block-scripts codegen",
```

Comment:
Yes – it's a necessary evil to avoid it running in CI. If possible we could bypass running `codegen` in `blocks/` so that an unresponsive URL does not tank the CI (since if we commit the files it will not be needed anyway)